t/002.upstream_dynamic.t

```diff
@@ -90,13 +92,13 @@ location = / {
 [200, 200, 999, 999, 200, 200, 999, 999]
 --- response_body eval
 ["Pass", "Pass", "Backup", "Backup", "Pass", "Pass", "Backup", "Backup"]
-=== TEST 4: Dynamic upstream with periodically failing DNS resolution and fallback specifying port number
+=== TEST 4: Dynamic upstream with periodically failing DNS resolution and specifying port number with strict fallback
 --- init
-`echo 'local-data: "example.com 1 A 127.0.0.2"' > /tmp/unbound_local_zone_ngx_upstream_jdomain.conf && unbound-control reload` or die $!;
+`echo 'local-data: "example.com 1 A 127.0.0.2"' > /tmp/unbound_local_zone_ngx_upstream_jdomain.conf && unbound-control reload ` or die $!;
 --- http_config
 resolver 127.0.0.88;
 upstream upstream_test {
-	jdomain example.com port=8000 retry_off fallback=127.0.0.3:12345;
+	jdomain example.com port=8000 retry_off fallback=127.0.0.3:12345 strict;
```

Comment:
For `Pass` tests, when we remove the DNS entry and reload unbound, it looks like the default response becomes NXDOMAIN, which basically means host not found, so we can confidently remove the `strict` from this test.

```suggestion
	jdomain example.com port=8000 retry_off fallback=127.0.0.3:12345;
```